src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxOperation.cs

```diff
@@ -79,7 +83,8 @@ public void Execute(TaskExecutionMode mode)
 
             try
             {
-                this.DacServices = new DacServices(this.ConnectionString);
+                // Pass in Azure authentication token if needed
+                this.DacServices = this.ConnInfo.ConnectionDetails.AzureAccountToken != null ? new DacServices(this.ConnectionString, new AccessTokenProvider(this.ConnInfo.ConnectionDetails.AzureAccountToken)) : new DacServices(this.ConnectionString);
```

Comment:
this.ConnInfo.ConnectionDetails is already checked for null in some way? if not please check that one too...

Reply:
added to constructor of DacFxOperation